src/legacy/core_plugins/kbn_doc_views/public/views/table/table.test.tsx

```diff
@@ -0,0 +1,250 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { mount } from 'enzyme';
+import { IndexPattern } from 'ui/index_patterns';
+// @ts-ignore
+import { findTestSubject } from '@elastic/eui/lib/test';
+// @ts-ignore
+import { flattenHitWrapper } from '../../../../data/public/index_patterns/index_patterns/flatten_hit';
+import { DocViewTable } from './table';
+
+// @ts-ignore
```

Comment:
nit: Not sure what's preferable, but you can achieve the same thing with `} as unknown as IndexPattern`

Reply:
when I'm using unknown, i get a typescript complaint the next line:

`indexPattern.flattenHit `

doing it the right way, would mean I'd to add more than 40 props. I've chosen the pragmatic way, but I'm open for improvements :)